package.json

```diff
@@ -4,20 +4,47 @@
   "author": "Tobias Koppers @sokra",
   "description": "file loader module for webpack",
   "files": [
-    "index.js"
+    "index.js",
+    "dist"
   ],
   "dependencies": {
     "loader-utils": "^1.0.2"
   },
   "devDependencies": {
-    "should": "~4.0.4",
+    "babel-cli": "^6.23.0",
+    "babel-jest": "^19.0.0",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-webpack": "^1.0.0",
+    "cross-env": "^3.2.3",
+    "del-cli": "^0.2.1",
+    "eslint": "^3.17.0",
+    "eslint-config-webpack": "^1.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "jest": "^19.0.2",
+    "lint-staged": "^3.3.1",
+    "nodemon": "^1.11.0",
+    "nsp": "^2.6.2",
+    "pre-commit": "^1.2.2",
     "standard-version": "^4.0.0",
-    "mocha": "~1.21.3"
+    "webpack-defaults": "^0.1.1"
   },
   "scripts": {
```

Comment:
Note that npm has a tendency to do its own sorting when it touches **package.json** somehow so I'm not sure if it's worthwhile to spend too much time on fixing this.